wolfcrypt/src/random.c

```diff
@@ -1102,20 +1102,20 @@ int wc_RNG_HealthTest_ex(int reseed, const byte* nonce, word32 nonceSz,
 }
 
 
-const byte seedA[] = {
+const FLASH_QUALIFIER byte _seedA[] = {
```

Comment:
@akmcomau , Is this section of changes still required even when using the `FLASH_QUALIFIER`? Is it possible to revert this section through line 1261? If not can you explain why using it dynamically allocated is required?

Reply:
This is because the `ConstantCompare()` and `wc_RNG_HealthTest()` are expecting the variables to be in SRAM, not the flash due to the parameters definition in the function.  These functions would have to be redefined to also include the `FLASH_QUALIFIER` in its parameters, which may not be an issue for `wc_RNG_HealthTest()`, but `ConstantCompare()` is called in other places, which would cause an issue.

I guess I could create a `ConstantCompareFlash()` function to achieve this, and conditionally call it instead.  And then read out the data from flash in that function and then call `ConstantCompare()`

Reply:
Okay that makes sense. I am okay with this change as-is.